lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -92,10 +92,11 @@ def self.create_backmerge_pr(token:, repository:, title:, head_branch:, base_bra
         intermediate_branch = "merge/#{head_branch.gsub('/', '-')}-into-#{base_branch.gsub('/', '-')}"
 
         if Fastlane::Helper::GitHelper.branch_exists_on_remote?(branch_name: intermediate_branch)
-          UI.user_error!("The intermediate branch `#{intermediate_branch}` already exists. Please check if there is an existing Pull Request that needs to be merged or closed first, or delete the branch.")
-          return nil
+          UI.important("An intermediate branch `#{intermediate_branch}` already existed on the remote. It will be deleted and any associated existing PR will be closed.")
```

Comment:
What do you think of making it clear that the closing of PRs is something GitHub does for us, not something we do ourselves e.g. via API calls?

```suggestion
          UI.important("An intermediate branch `#{intermediate_branch}` already exists on the remote. It will be deleted and GitHub will close any associated existing PR.")
```

Also suggesting to use "exists" over "existed" because the latter made me think the branch no longer existed at the time the message was printed, but that's not the case given the delete call happens afterwards.